### PR TITLE
Dark Mode

### PR DESCRIPTION
**Pull request by pacocoursey**
## Description

Adds support for dark mode styling 🕶 

- Automatically reads from OS-level setting (`prefers-color-scheme: dark`)
- Toggle-able with <kbd>Alt+T</kbd>
- After toggling, the preference is saved in localStorage for next time (overrides OS-level)

Styling is done via CSS variables where possible. Color is hard-coded in one place (the canvas grid dot color). This place could be updated to automatically read from CSS variables via `getComputedStyle` and `getPropertyValue`, if you wanted.

## Motivation and Context

Manifest is beautiful, would love if it didn't hurt my eyes at night.

## Screenshots (if appropriate):

localhost_3000_ (1)

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jonathontoon** [comment]
This looks awesome @pacocoursey! I'm just about to head out somewhere, so will take a look at it later tonight. ✌️

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 - [Acknowledgements](#acknowledgements)
 
 ## Introduction
-Manifest is a grid-based pinboard for note taking. Simply click and drag anywhere to create a note and snap it to the grid set by your current window size. All notes can be be moved, resized and deleted. Your browser's local storage is also utilized to save notes between sessions and while offline.
```
Comment (jonathontoon) [inline README.md]: 🙈

File: src/js/index.js [hunk 2]
```diff
@@ -396,6 +397,47 @@ function handleBoardDragEnd(e) {
   App Functions
 */
 
+function toggleTheme() {
+  if (theme === "light") {
+    document.body.className = theme = "dark";
+    setLocalStorageItem("theme", "dark");
```
Comment (jonathontoon) [inline src/js/index.js]: Would you mind if we prefixed the local storage key with _manifest__? Helps keep in line with the memos key.

File: src/js/index.js [hunk 3]
```diff
@@ -396,6 +397,47 @@ function handleBoardDragEnd(e) {
...
+    } else {
+      document.body.className = "";
+      theme = "light";
+      setLocalStorageItem("theme", "light");
+    }
+    return;
+  }
+
+  if (window.matchMedia("(prefers-color-scheme: dark)").matches) {
+    document.body.className = theme = "dark";
```
Comment (jonathontoon) [inline src/js/index.js]: So far I've been using classList.add/remove. Just worried if we don't use that method, things might break with this approach if <body> has additional classes appended to it in the future.

**Review by jonathontoon — CHANGES_REQUESTED** [review]
Just two small changes added in the comments. Otherwise this looks really rad. Can't wait to use it.

Thanks friend! 🙏

File: src/js/index.js (hunk 3 above)
Comment (pacocoursey) [reply src/js/index.js]: Great! I actually prefer that too, should have checked if you were already using it :)

**Comment by pacocoursey** [comment]
All done @jonathontoon!

**Review by jonathontoon — APPROVED** [review]
Great stuff. Until I've got it up on master you can test it live at staging.manifest.app.

**Pull request merged**